src/librustdoc/formats/renderer.rs

```diff
@@ -28,7 +29,8 @@ pub(crate) trait FormatRenderer<'tcx>: Sized {
     ) -> Result<(Self, clean::Crate), Error>;
 
     /// Make a new renderer to render a child of the item currently being rendered.
-    fn make_child_renderer(&self) -> Self;
+    fn make_child_renderer(&mut self) -> Self::InfoType;
```

Comment:
Could these be documented? Same with `InfoType`. (At the very least the old doc-comment should be removed as it's no longer correct)

Reply:
Once I have the definitive API ready, I'll definitely add documentation. 👍

Reply:
Documentation above still needs changing. Also: I don't think this method-name accuratly describes what it does anymore. Maybe something like `save_modinfo`?